code/controllers/subsystem/movement/movement_types.dm

```diff
@@ -113,11 +113,11 @@
 		return
 
 	var/visual_delay = controller.visual_delay
-	var/success = move()
+	var/result = move() //Is not necesarily a bool, can also return MOVELOOP_NOT_READ
 
 	SEND_SIGNAL(src, COMSIG_MOVELOOP_POSTPROCESS, success, delay * visual_delay)
```

Comment:
wrong var here

Reply:
thanks, must've missed that while bringing over the changes